src/olympia/amo/cron.py

```diff
@@ -159,28 +159,6 @@ def collection_subscribers():
 
 
 @cronjobs.register
-def unconfirmed():
```

Comment:
I'd give them a grace-period of a month or so and then delete them all together and remove `confirmationcode` afterwards. Does that work?